### PR TITLE
#162162971 Users should be able to Like and Dislike An Article

### PR DESCRIPTION
**Pull request by kizzanaome**
### What does this PR do?
- Enable user to like or dislike an article.

### Description of Task to be completed?
- Create LikeDislike component
- Add Like and Dislike icons to the Article component

### How should this be manually tested?
- Clone the repository by running git clone https://github.com/andela/ah-frontend-poseidon.git
- Run git checkout  ft-dislike-like-article-162162971
- Start the server by running yarn start
- Navigate to the articles page, and click ReadMore, to display an article.
- You will be able to like/dislike that article

### What are the relevant pivotal tracker stories?
#162162971

### Screenshot

image

### CODE REVIEW

**Review by billkabanga — APPROVED** [review]

**Review by katunold — APPROVED** [review]

**Review by arthurarty — APPROVED** [review]

File: src/components/articles/LikeDislikeComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  render() {
+    const { article, likeColor, dislikeColor, onLike, ondislike } = this.props;
+    return (
+      <div className="container">
+        <div className="container" />
+        <div>
+          {article.likes}
+          {this.renderThumb(
+            'thumbs-up',
+            { color: likeColor, width: '80px', height: '30px' },
```
Comment (arthurarty) [inline src/components/articles/LikeDislikeComponent.jsx]: @kizzanaome I noticed styling on line 26 which is very similar to they styling online 32. What do you think of having a css class that has this css and then adding the class to both icons.

**Review by arthurarty — COMMENTED** [review]
@kizzanaome What you think of having a css class to handle your styling for the like and disklike button.

File: src/components/articles/LikeDislikeComponent.jsx (hunk 1 above)
Comment (kizzanaome) [reply src/components/articles/LikeDislikeComponent.jsx]: Okay, Thanks for pointing that out let me fix it.

Comment (kizzanaome) [reply src/components/articles/LikeDislikeComponent.jsx]: I have fixed this

**Review by Luleherll — CHANGES_REQUESTED** [review]
I have tested the feature and I think the page should not reload when a user likes because we would be resetting the state. Please look through @katunold and @m-asiimwe's implementation.

**Comment by kizzanaome** [comment]
@Luleherl l  sorted that issue already.

**Comment by Luleherll** [comment]
> @Luleherl l sorted that issue already.

I first pulled before I tested, maybe you had not pushed the code. Am going to pull again and test it out.

**Review by Luleherll — APPROVED** [review]

**Comment by Luleherll** [comment]
Nice work @kizzanaome. Please also squash the commits.

**Pull request merged**